jwst/lib/suffix.py

```diff
@@ -47,6 +47,7 @@
     's2d', 's3d', 'snr',
     'uncal',
     'wfscmb', 'whtlt',
+    'undersampling_correction',
```

Comment:
I believe this entry needs to go into the calculated suffixes list starting down at line 60. This list (SUFFIXES_TO_ADD) is the list of standard suffixes for archived products, whereas the list below is for all the individual steps.